### ISSUE TITLE
Incompatibility with Xcode 11

### ISSUE THREAD

**Issue by youssmak** [voxeet/voxeet-uxkit-cordova#18]
**Describe the bug**

I'm unable to build since i have upgrade Xcode to the last version 

```
*** Downloading voxeet-ios-sdk.framework binary at "1.4.2"
***  Skipped installing voxeet-ios-sdk.framework binary due to the error:
	"Incompatible Swift version - framework was built with 5.0.1 (swiftlang-1001.0.82.4 clang-1001.0.46.5) and the local version is 5.1 (swiftlang-1100.0.270.13 clang-1100.0.33.7)."
```
Could you please update the ios sdk dependency to avoid Incompatible Swift version ?

**Comment by VTCoco** [comment]
Working on it to bump Voxeet SDKs versions :)

**Comment by VTCoco** [comment]
It should be available with 1.2.14, can you try on your side @youssmak please ?

**Comment by youssmak** [comment]
fixed, thank you !
